microsoft/CBL-Mariner
# Add python-resolvelib package to support ansible-galaxy

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*

---

###### Summary 

What does the PR accomplish, why was it needed?
The ansible spec file within Mariner is missing a dependency on `python-resolvelib` for the ansible-galaxy command set. This pr adds and builds this python package within Mariner.

###### Change Log  



- Add python-resolvelib package to Mariner
- Add runtime requires on python-resolvelib to ansible spec 

###### Does this affect the toolchain?  



**NO**

###### Test Methodology

- local build
- install ansible rpm in mariner container with resolvelib package
- check failing command `ansible-galaxy --help` post install for errors
